modules/virt-importing-vm-wizard.adoc

```diff
@@ -82,7 +85,7 @@ endif::[]
 . Click *Edit* to update the following settings:
 
 ifdef::virt-importing-rhv-vm[]
-* *General* -> *Name*: The VM name is limited to 63 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
+* *General* -> *Name*: The VM name is limited to 63 characters.
```

Comment:
I think something has broken the numbering of the procedure:
https://cnv-import-prereq-reorg--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html#virt-importing-vm-prerequisites_virt-importing-rhv-vm
Looks like step 11 is built as step 1 - ?

Reply:
Fixed